app/usecase/repo/repotest/availablekey.go

```diff
@@ -11,35 +11,47 @@ import (
 var _ repo.AvailableKey = (*AvailableKeyFake)(nil)
 
 type AvailableKeyFake struct {
-	keys map[entity.Key]bool
+	keys map[entity.Key]struct{}
```

Comment:
```suggestion
	keys map[entity.Key]struct{}
```
Is this for space optimization?

Reply:
Yes, it should save some space https://dave.cheney.net/2014/03/25/the-empty-struct